sw/host/rom_ext_image_signer/vendor-dependencies.sh

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
+# Copyright lowRISC contributors.
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+
+set -e
+
+if [ "$#" != "1" ]; then
+    printf "\nIncorrect usage:\n" >&2
+    printf "Usage: $(basename "$0") add|remove\n" >&2
+    exit 1
+fi
+
+rm -rf vendored_dependencies
+
+if [ "$1" == "add" ]; then
+    mkdir vendored_dependencies
+
+    # TODO - do we need any?
+    VENDOR_FLAGS=
```

Comment:
We want `--locked`, so we always pull the versions specified in Cargo.lock, rather than any versions that satisfy the constraints in Cargo.toml.